core/src/main/groovy/com/novoda/gradle/release/AndroidLibrary.groovy

```diff
@@ -24,9 +24,13 @@ class AndroidLibrary implements SoftwareComponentInternal {
     AndroidLibrary(Project project) {
         ObjectFactory objectFactory = project.getObjects()
 
-        def newVersion = GradleVersion.current() > GradleVersion.version("4.1")
-        Usage api = objectFactory.named(Usage.class, newVersion ? Usage.JAVA_API : "for compile")
-        Usage runtime = objectFactory.named(Usage.class, newVersion ? Usage.JAVA_RUNTIME : "for runtime")
+        // Using the new Usage in 4.1 will make the plugin crash
+        // as comparing logic is still using the old Usage.
+        // 4.1 : https://git.io/vFAnQ
+        // 4.2 : https://git.io/vFAnd
```

Comment:
better include the unshortened links here which give an idea where they points to.

Reply:
What do you think @StefMa? Personally, I agree with @passsy but as you asked me to use git.io if you agree, I will make a new commit.